src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -490,35 +453,21 @@ protected function setup_messages( array $events ) {
 			return;
 		}
 
-		$tax = $this->context->get('taxonomy');
-
-		// @todo move this to Filterbar @stephen
-		$filters = array_filter( (array) $this->context->get( 'view_data' ) );
+		$fast_forward_link  = $this->get_fast_forward_link( true );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: f0b461bfd9157c73aac2aff782b5b8c2</sub>